workflow

```diff
@@ -1,373 +1,375 @@
-
-bpy.context.scene.render.resolution_x = 1080
-bpy.context.scene.render.resolution_y = 1920
-
-bpy.data.objects['Camera'].select_set(True)
-
-bpy.data.scenes[0].camera = bpy.data.objects['Camera']
-
-# render
-bpy.data.worlds["World"].node_tree.nodes["Background"].inputs[0].default_value = (0., 0., 0., 1) #background escuro
-
-bpy.context.scene.frame_end = last_frame + 10
-bpy.context.scene.eevee.use_bloom = True
-bpy.context.scene.eevee.use_gtao = True
-bpy.context.scene.eevee.use_ssr = True
-bpy.context.scene.eevee.use_motion_blur = True
+            make_curve(points, loop2)
+
+            # Handle materials
+            insert_material(loop, loop2)    
+
+            animate(time_initial, time_final, loop2) 
+
+            if time_final > last_frame: # Finding the last frame
+                last_frame = time_final
+
+    bpy.context.scene.frame_current = 0 # Return to initial frame
+
+    # Light
+    bpy.ops.object.light_add(type='SUN', align='WORLD', location=(0, -5.04, 4.6), rotation=(1.05418, 0, 0))
+
+    # Camera
+    bpy.ops.object.camera_add(enter_editmode=False, align='VIEW', location=(-1.0, -5.0, 1.5), rotation=(87.3975 / 57.3, 0, -12.974 / 57.3))
+
+    bpy.context.scene.render.resolution_x = 1080
+    bpy.context.scene.render.resolution_y = 1920
+
+    bpy.data.objects['Camera'].select_set(True)
+
+    bpy.data.scenes[0].camera = bpy.data.objects['Camera']
+
+    # Render
+    bpy.data.worlds["World"].node_tree.nodes["Background"].inputs[0].default_value = (0.0, 0.0, 0.0, 1) # Dark background
+
+    bpy.context.scene.frame_end = last_frame + 10
+    bpy.context.scene.eevee.use_bloom = True
+    bpy.context.scene.eevee.use_gtao = True
+    bpy.context.scene.eevee.use_ssr = True
+    bpy.context.scene.eevee.use_motion_blur = True
+
+    # Save
+    bpy.ops.wm.save_mainfile(filepath=os.getcwd() + "output.blend") # Save as output.blend
+    print(f"Final frame: {last_frame}")
+    print('Took {} minutes'.format((time.time() - start_time) / 60))
+
+# MAIN
+print("Hello, welcome to the simulation and animation workflow of atmospheric showers.")
+flag = True
+while flag:
+  answer = input("\nPlease indicate if you already have CORSIKA installed on your machine (yes/no/quit): ")
+  if answer == "yes":
+    flag_2 = True
+    while flag_2:
+      answer_2 = input("\nTo work correctly, this file must be executed within the 'run' folder of CORSIKA. Are you at 'run' folder? (yes/no/quit): ")
+      if answer_2 == "yes":
+        print("\nLet's start with running the simulation.")
+        print("\nGenerating the DAT files...")
+        command_DAT_files = "./corsika77500Linux_EPOS_urqmd < all-inputs-epos"
+        generate_DAT_files(command_DAT_files)
+        print("\nConverting DAT files to TXT format...")
+        command_tracks2root = "perl tracks2root.pl"
+        tracks2root(command_tracks2root)
+        flag_2 = False
+        flag = False
+      elif answer_2 == "no":
```

Comment:
knowing the function of answer_2 will facilitate the reading of the code at this point